content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -350,6 +350,13 @@ users:
       client-key: /path/to/key.pem          # key matching the cert
 ```
 
+#### Enable the Admission Control Plugin(s)
+After doing above configuration open the `kube-apiserver` manifest for editing and in the `--enable-admissions-plugin` line under command, add the image policy webhook to the list:
+
+```
+--enable-admissions-plugins=NodeRestriction,ImagePolicyWebhook
+```
```

Comment:
I'm afraid that this doesn't look right. Why should we make this specific change?

Reply:
because this is a mandatory change required to enable external imagepolicywebhook scannner connection from kube-apiserver. Till the time we dont enable this plugins admission controller wont be able to redirect incoming request to imagepolicywebhook

Reply:
I have tried this and if we dont do this then it wont work

Reply:
This page that you're updating is a reference page. If you want to, you can add a new _separate_ task page about configuring container image policy webhooks. It'd be a welcome improvement. 

That new page would go inside https://kubernetes.io/docs/tasks/

Reply:
but this is actually a part of this reference documentation itself but anyways I will create this task and can I add the hyperlink here in same section because for imagepolicywebhook that would complete steps

Reply:
this is actually but part of imagepolicywebhook which I saw missing in main kubernetes documentation 
--enable-admissions-plugin=ImagePolicyWebhook